backend/py-files/dual_print.py

```diff
@@ -4,7 +4,7 @@
 def dual_print(massage, level="info"):
```

Comment:
message? :P

Reply:
Do you want some massage?
btw its old :( sorry

Reply:
ohhhh yeaaaa babyy. put some oil.